components/eam/src/physics/crm/crm_history.F90

```diff
@@ -361,11 +355,6 @@ subroutine crm_history_init(species_class)
       call add_default('MMF_NR    ', 1, ' ')
    end if
 
-#if defined(MMF_MOMENTUM_FEEDBACK) || defined(MMF_ESMT)
-   call add_default ('MMF_DU', 1, ' ')
```

Comment:
This determines whether they go into the monthly h0 file by default. I figured this isn't necessary since we rarely look at these variables unless we are focused on analyzing the momentum budget, which would require a careful consideration of more output variables than this, so it made sense to me not to include them by default.